### PR TITLE
Initialize Flexbox Layout for NavBar, Page, and Footer

### PR DESCRIPTION
**Pull request by imartinez921**
_See below where NavBar: blue, Page: yellow, Footer: purple_

image

### CODE REVIEW

File: frontend/components/nav_bar/nav_bar_main.jsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ class NavBarMain extends React.Component {
 
                     {this.state.menuOpen ? dropMenu : null}
                 </nav>
```
Comment (k-shan) [inline frontend/components/nav_bar/nav_bar_main.jsx:65]: Feel free to ignore if you're going to address this later, I'm new to `nav` elements, are they okay to be nested?

File: frontend/components/page/page.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    history,
+    params,
+    path,
+    currentUser,
+    errors,
+    clearPlaylistErrors,
+}) => {
+    return (
+        <div className="page_container">
+                <NavBarMainContainer history={history} />
```
Comment (k-shan) [inline frontend/components/page/page.jsx:18]: nit: indent, run prettier please

File: frontend/components/page/page.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                <div className="current-container">
+                    <CurrentViewContainer
+                        currentUser={currentUser}
+                        params={params}
+                        path={path}
+                        history={history}
+                    />
+                </div>
+            </div>
+                <Footer
```
Comment (k-shan) [inline frontend/components/page/page.jsx:35]: nit: Indent

File: frontend/components/page/page.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                    currentUser={currentUser}
+                    params={params}
+                    path={path}
+                    history={history}
```
Comment (k-shan) [inline frontend/components/page/page.jsx:39]: It's surprising all the components need all the same props. No need to change, but as you refactor if you find you don't need a prop, please remove it.

File: frontend/components/nav_bar/nav_bar_main.jsx (hunk 1 above)
Comment (imartinez921) [reply frontend/components/nav_bar/nav_bar_main.jsx:65]: I believe they are? As far as I know (which isn't much), they are just like divs except are better tags for accessibility - [Semantic HTML elements](https://www.w3schools.com/html/html5_semantic_elements.asp)

File: app/assets/stylesheets/05-page.scss [hunk 5]
```diff
@@ -0,0 +1,35 @@
+.page_container {
```
Comment (k-shan) [inline app/assets/stylesheets/05-page.scss:1]: What's the significance of the name `05-page`?

File: app/assets/stylesheets/06-home.scss [hunk 6]
```diff
@@ -1,13 +1,13 @@
 .home {
     background-color: rgba(0, 0, 0, 0.6);
-    height: calc(100% - 60px);
-    width: calc(100% - 60px);
+    height: 100%;
+    width: 100%;
```
Comment (k-shan) [inline app/assets/stylesheets/06-home.scss:4]: Nice

File: frontend/components/app.jsx [hunk 7]
```diff
@@ -6,29 +6,29 @@ import { LoggedoutRoute } from "../util/session_route_util";
 import SplashContainer from "./splash/splash_container";
 import SignupFormContainer from "./session/signup_form_container";
 import LoginFormContainer from "./session/login_form_container";
-import MainContainer from "./main/main_container";
+import PageContainer from "./page/page_container";
 // import EditPlaylistContainer from './playlists/edit_playlist_container'
```
Comment (k-shan) [inline frontend/components/app.jsx:10]: nit: maybe uncomment until you need it

File: app/assets/stylesheets/05-page.scss (hunk 5 above)
Comment (imartinez921) [reply app/assets/stylesheets/05-page.scss:1]: In this version of Rails, there is a SASS compiler that runs the .scss all together as the CSS sheets. We were taught to create a new .scss file per component to help with organization + number it for ordering.

**Pull request merged**